src/components/ToDoComponent/Card.tsx

```diff
@@ -0,0 +1,112 @@
+import { Image, Text, View, TouchableOpacity } from 'react-native';
+import React, { useState } from 'react';
+import Animated, {
+  runOnJS,
+  useAnimatedGestureHandler,
+  useAnimatedStyle,
+  useSharedValue,
+  withTiming,
+} from 'react-native-reanimated';
+import {
+  GestureHandlerRootView,
+  PanGestureHandler,
+} from 'react-native-gesture-handler';
+import Images from '../../constants/images/Image';
+import Toast from 'react-native-toast-message';
+import { CardStyles } from './Styles/CardStyles';
+
+const Card = ({ item, posStyle, changecard, removeCard }: any) => {
```

Comment:
```suggestion
const Card = ({ item, posStyle, changecard, removeCard }: any) => {

instead of `any` we can create a type for the props please refer here https://github.com/Real-Dev-Squad/mobile-app/blob/8ae1eae1312e44c458495266fca1bd27c2995b76/src/components/GoalCard/index.tsx
```

Reply:
will do that